src/components/Popup.js

```diff
@@ -1,9 +1,11 @@
 export default class Popup {
   constructor(popupSelector) {
     this.modal = document.querySelector(popupSelector);
+    this.currentId = "";
+    this.removeHandler = "";
```

Comment:
Needs correction: the Popup should not know anything about the delete confirmation